htdocs/themes/berlin_mcjc/people/items/show.php

```diff
@@ -19,8 +19,9 @@
 }
 ?>
 <div class="primary person<?php echo $itemClasses; ?>">
+<div class="person title">
```

Comment:
does this need to be in a div or can the class just go right on the <h1> element?